test/hmactest.c

```diff
@@ -7,6 +7,12 @@
  * https://www.openssl.org/source/license.html
  */
 
+/*
+ * HMAC low level APIs are deprecated for public use, but still ok for internal
+ * use.
+ */
+#include "internal/deprecated.h"
+
```

Comment:
Is there any point in keeping this program?  Replace it with stanzas for evp_test?

Reply:
It is testing the low level API.  There exist `evp_test` tests already.  I've not checked the overlap.

I'm fine to remove it.

Reply:
How does this differ from #10831?

Reply:
Going through evp_test still ends up testing the low level API, doesn't it?

Reply:
#10831 doesn't exercise the low level APIs, it's EVP all the way.

The functional is tested either way but the APIs not necessarily (at least in the future).